lib/task-data/schemas/dell-wsman-reset-components.json

```diff
@@ -0,0 +1,20 @@
+{
+    "copyright": "Copyright 2017, Dell EMC, Inc.",
+    "definitions": {
+        "components": {
+            "type": "array",
+            "description": "A list of components that will be reset",
+            "items": {
+                "type": "string",
```

Comment:
Use emum type could be better than string.
The parameter could only be bios, diag, drvpack, idrac and lcdata

Reply:
The component name is case-insensitive, it could be BiOs.